packages/bpk-component-badge/examples.js

```diff
@@ -0,0 +1,187 @@
+);
+
+const CenteredExample = () => (
+  <BadgeLayout>
+    <div>
+      The badge is aligned to the centre of this text.{' '}
+      <BpkBadge centered>Centered</BpkBadge>
+    </div>
+  </BadgeLayout>
+);
+
+const DockedLeadingExample = () => (
+  <BadgeLayout docked="left">
+    <BpkBadge docked="left">Advert</BpkBadge>
+  </BadgeLayout>
+);
+
+const DockedTrailingExample = () => (
+  <BadgeLayout docked="right">
+    <BpkBadge docked="right">Advert</BpkBadge>
+  </BadgeLayout>
+);
+
+const WarningExample = () => (
+  <BadgeLayout>
+    <BpkBadge type={BADGE_TYPES.warning}>Warning</BpkBadge>
+    &nbsp;
+    <BpkBadge type={BADGE_TYPES.warning}>
+      <BpkSmallTickIcon /> &nbsp;Warning
+    </BpkBadge>
+    &nbsp;
+    <BpkBadge type={BADGE_TYPES.warning}>
+      Warning&nbsp; <BpkSmallTickIcon />
+    </BpkBadge>
+  </BadgeLayout>
+);
+
+const SuccessExample = () => (
+  <BadgeLayout>
+    <BpkBadge type={BADGE_TYPES.success}>Success</BpkBadge>
+    &nbsp;
+    <BpkBadge type={BADGE_TYPES.success}>
+      <BpkSmallTickIcon />
+      &nbsp;Success
+    </BpkBadge>
+    &nbsp;
+    <BpkBadge type={BADGE_TYPES.success}>
+      Success&nbsp;
+      <BpkSmallTickIcon />
+    </BpkBadge>
+  </BadgeLayout>
+);
+
+const DestructiveExample = () => (
+  <BadgeLayout>
+    <BpkBadge type={BADGE_TYPES.destructive}>Destructive</BpkBadge>
+    &nbsp;
+    <BpkBadge type={BADGE_TYPES.destructive}>
+      <BpkSmallCloseIcon />
+      &nbsp;Destructive
+    </BpkBadge>
+    &nbsp;
+    <BpkBadge type={BADGE_TYPES.destructive}>
+      Destructive&nbsp;
+      <BpkSmallCloseIcon />
+    </BpkBadge>
+  </BadgeLayout>
+);
+
+const LightExample = () => (
+  <BadgeLayout>
+    <BpkBadge type={BADGE_TYPES.light}>Light</BpkBadge>
+    &nbsp;
+    <BpkBadge type={BADGE_TYPES.light}>
+      <BpkSmallTickIcon />
+      &nbsp;Light
+    </BpkBadge>
+    &nbsp;
+    <BpkBadge type={BADGE_TYPES.light}>
+      Light&nbsp;
+      <BpkSmallTickIcon />
+    </BpkBadge>
+  </BadgeLayout>
+);
+
+const InverseExample = () => (
+  <BadgeLayout>
+    <BpkBadge type={BADGE_TYPES.inverse}>Inverse</BpkBadge>
+    &nbsp;
+    <BpkBadge type={BADGE_TYPES.inverse}>
+      <BpkSmallTickIcon />
+      &nbsp;Inverse
+    </BpkBadge>
+    &nbsp;
+    <BpkBadge type={BADGE_TYPES.inverse}>
+      Inverse&nbsp;
+      <BpkSmallTickIcon />
+    </BpkBadge>
+  </BadgeLayout>
+);
+
+const OutlineExample = () => (
+  <BadgeLayout>
+    <BpkBadge type={BADGE_TYPES.outline}>Outline</BpkBadge>
+    &nbsp;
+    <BpkBadge type={BADGE_TYPES.outline}>
+      <BpkSmallTickIcon />
+      &nbsp;Outline
+    </BpkBadge>
+    &nbsp;
+    <BpkBadge type={BADGE_TYPES.outline}>
+      Outline&nbsp; <BpkSmallTickIcon />
+    </BpkBadge>
+  </BadgeLayout>
+);
+
+export {
+  DefaultExample,
+  WithIconsExample,
+  CenteredExample,
+  DockedLeadingExample,
+  DockedTrailingExample,
+  WarningExample,
+  SuccessExample,
+  DestructiveExample,
+  LightExample,
+  InverseExample,
+  OutlineExample,
+};
```

Comment:
Why don't we put the `export` keyword inline?
Like `export const OulineExample...`

Reply:
I've just always preferred the neatness of doing it this way, I prefer it